### PR TITLE
docs: resctructure to be in compliance according to Grafana Plugin Re…

### PR DESCRIPTION
**Pull request by miguel91it**
docs: resctructure to be in compliance according to Grafana Plugin Re…

Grafana has a tool named Plugin Validator https://github.com/grafana/plugin-validator which tells us what is wrong in the plugin before we send to be published.

docs: fix some images links to point to the src/img/assets directory instead of public links on the internet

docs: adjust the title of the README

### CODE REVIEW

**Comment by miguel91it** [comment]
@alvarowolfx, fixed. Can we merge to develop?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,3 +97,156 @@ The content of the commit message body should contain:
...
+### Run the Grafana
+
+1. Start the docker container
+
+   ```bash
+    docker run \
+      --network="host" -e "GF_LOG_MODE=console file" \
+      -e "GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=golioth-websocket-datasource" \
+      -p 3000:3000 \
+      -v ~/Golioth/Codes/grafana-plugins:/var/lib/grafana/plugins \
```
Comment (alvarowolfx) [inline CONTRIBUTING.md:193]: this probably need to be changed to `path/to/your/clone`

**Review by alvarowolfx — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (miguel91it) [reply CONTRIBUTING.md]: Now it's ok.

**Review by alvarowolfx — APPROVED** [review]
LGTM

**Comment by miguel91it** [comment]
@alvarowolfx, i needed to make a new rebase because there were another PR which was concluded first. And because of that, this branch had to be rebased and some conflicts to solve. I finished everything and it's all on the remote now (rebased and conflicts sovled)

**Comment by miguel91it** [comment]
I can finish the PR or you need to review again?

**Pull request merged**